src/core/installAddon.js

```diff
@@ -416,12 +422,28 @@ export class WithInstallHelpers extends React.Component {
       });
   }
 
-  enable() {
-    const { _addonManager, dispatch, guid, iconUrl, name } = this.props;
+  enable({ sendTrackingEvent = true } = {}) {
+    const {
+      _addonManager,
+      _tracking,
+      dispatch,
+      guid,
+      iconUrl,
+      name,
+      type,
+    } = this.props;
 
     return _addonManager
       .enable(guid)
       .then(() => {
+        if (sendTrackingEvent) {
```

Comment:
I don't see this being called/sent on disco. shouldn't it be?

Reply:
mmh, how? I tried with a theme and an extension.

Reply:
oh? does it work for you?

Reply:
Yes

Reply:
hmm that's weird. I see it now too :/ ok cool : )